src/vulkan/rt64_vulkan.cpp

```diff
@@ -3953,11 +4000,42 @@ namespace RT64 {
         createInfo.enabledLayerCount = 0;
 
         // Check for extensions.
-        uint32_t extensionCount;
-        vkEnumerateInstanceExtensionProperties(nullptr, &extensionCount, nullptr);
+        const char** extensionNames = nullptr;
```

Comment:
I had the impression that it was Mupen itself that initialized the Vulkan Instance under the new API. Can you confirm if that is the case?

Reply:
The plugin has to create the vulkan instance and pass it to the front-end, the documentation could be slightly more clear on this though.